Storybookで"@/"から始まる絶対パスでモジュールを解決できない

# 概要
`ModuleNotFoundError: Module not found: Error: Can't resolve '@/libs/twmerge/index' in '/home/reohakuta/repos/shikosai32-front/src/components/Button'`

参考: [Storybookでimport aliasを使う](https://tamalog.szmd.jp/storybook-absolute-imports/)

# エラー文
```shell
reohakuta in shikosai32-front on   feat/17-add_button  📝 modified  🤷 untracked
❯  yarn run sb
yarn run v1.22.17
$ start-storybook -p 6006
info @storybook/react v6.5.10
info
info => Loading presets
info => Serving static files from ././public at /
info Addon-docs: using MDX1
info => Using PostCSS preset with postcss@8.4.16
info => Using default Webpack5 setup
<i> [webpack-dev-middleware] wait until bundle finished
10% building 0/1 entries 0/0 dependencies 0/0 modules
info => Using cached manager
21% building 3/15 entries 2052/2093 dependencies 233/485 modules
🌼 daisyUI components 2.24.0  https://github.com/saadeghi/daisyui
  ✔ Including:  base, components, themes[1], utilities

70% sealing plugins DocGenPluginDeprecationWarning: 'createIdentifier' has been deprecated since v4.0.0. Use the appropriate method on 'ts.factory' or the 'factory' supplied by your transformation context instead.
DeprecationWarning: '' has been deprecated since v4.0.0. Use the appropriate method on 'ts.factory' or the 'factory' supplied by your transformation context instead.
DeprecationWarning: 'createLiteral' has been deprecated since v4.0.0. Use `factory.createStringLiteral`, `factory.createStringLiteralFromNode`, `factory.createNumericLiteral`, `factory.createBigIntLiteral`, `factory.createTrue`, `factory.createFalse`, or the factory supplied by your transformation context instead.
DeprecationWarning: 'createBinaryExpression' has been deprecated since v4.0.0. Use the appropriate method on 'ts.factory' or the 'factory' supplied by your transformation context instead.
DeprecationWarning: 'createExpressionStatement' has been deprecated since v4.0.0. Use the appropriate method on 'ts.factory' or the 'factory' supplied by your transformation context instead.
DeprecationWarning: 'createPropertyAssignment' has been deprecated since v4.0.0. Use the appropriate method on 'ts.factory' or the 'factory' supplied by your transformation context instead.
DeprecationWarning: 'createObjectLiteralExpression' has been deprecated since v4.0.0. Use the appropriate method on 'ts.factory' or the 'factory' supplied by your transformation context instead.
DeprecationWarning: 'createExpressionStatement' has been deprecated since v4.0.0. Use the appropriate method on 'ts.factory' or the 'factory' supplied by your transformation context instead.
DeprecationWarning: 'createTypeOfExpression' has been deprecated since v4.0.0. Use the appropriate method on 'ts.factory' or the 'factory' supplied by your transformation context instead.
DeprecationWarning: 'createElementAccessExpression' has been deprecated since v4.0.0. Use the appropriate method on 'ts.factory' or the 'factory' supplied by your transformation context instead.
DeprecationWarning: 'createIfStatement' has been deprecated since v4.0.0. Use the appropriate method on 'ts.factory' or the 'factory' supplied by your transformation context instead.
DeprecationWarning: 'createBlock' has been deprecated since v4.0.0. Use the appropriate method on 'ts.factory' or the 'factory' supplied by your transformation context instead.
DeprecationWarning: 'createVariableDeclaration' has been deprecated since v4.0.0. Use the appropriate method on 'ts.factory' or the 'factory' supplied by your transformation context instead.
DeprecationWarning: 'createCatchClause' has been deprecated since v4.0.0. Use the appropriate method on 'ts.factory' or the 'factory' supplied by your transformation context instead.
DeprecationWarning: 'createTryStatement' has been deprecated since v4.0.0. Use the appropriate method on 'ts.factory' or the 'factory' supplied by your transformation context instead.
DeprecationWarning: 'createFalse' has been deprecated since v4.0.0. Use the appropriate method on 'ts.factory' or the 'factory' supplied by your transformation context instead.
99% done plugins webpack-hot-middlewarewebpack built preview 87584c4da0edd89a2389 in 9314ms
ModuleNotFoundError: Module not found: Error: Can't resolve '@/libs/twmerge/index' in '/home/reohakuta/repos/shikosai32-front/src/components/Button'
    at /home/reohakuta/repos/shikosai32-front/node_modules/webpack/lib/Compilation.js:2016:28
    at /home/reohakuta/repos/shikosai32-front/node_modules/webpack/lib/NormalModuleFactory.js:798:13
    at eval (eval at create (/home/reohakuta/repos/shikosai32-front/node_modules/webpack/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:10:1)
    at /home/reohakuta/repos/shikosai32-front/node_modules/webpack/lib/NormalModuleFactory.js:270:22
    at eval (eval at create (/home/reohakuta/repos/shikosai32-front/node_modules/webpack/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:9:1)
    at /home/reohakuta/repos/shikosai32-front/node_modules/webpack/lib/NormalModuleFactory.js:434:22
    at /home/reohakuta/repos/shikosai32-front/node_modules/webpack/lib/NormalModuleFactory.js:116:11
    at /home/reohakuta/repos/shikosai32-front/node_modules/webpack/lib/NormalModuleFactory.js:670:25
    at /home/reohakuta/repos/shikosai32-front/node_modules/webpack/lib/NormalModuleFactory.js:855:8
    at /home/reohakuta/repos/shikosai32-front/node_modules/webpack/lib/NormalModuleFactory.js:975:5
resolve '@/libs/twmerge/index' in '/home/reohakuta/repos/shikosai32-front/src/components/Button'
  Parsed request is a module
  using description file: /home/reohakuta/repos/shikosai32-front/package.json (relative path: ./src/components/Button)
    Field 'browser' doesn't contain a valid alias configuration
    resolve as module
      /home/reohakuta/repos/shikosai32-front/src/components/Button/node_modules doesn't exist or is not a directory
      /home/reohakuta/repos/shikosai32-front/src/components/node_modules doesn't exist or is not a directory
      /home/reohakuta/repos/shikosai32-front/src/node_modules doesn't exist or is not a directory
      looking for modules in /home/reohakuta/repos/shikosai32-front/node_modules
        /home/reohakuta/repos/shikosai32-front/node_modules/@/libs doesn't exist
      /home/reohakuta/repos/node_modules doesn't exist or is not a directory
      /home/reohakuta/node_modules doesn't exist or is not a directory
      /home/node_modules doesn't exist or is not a directory
      /node_modules doesn't exist or is not a directory

WARN Broken build, fix the error above.
WARN You may need to refresh the browser.

info => Loading presets
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

--- a/.storybook/main.js
+++ b/.storybook/main.js
@@ -2,7 +2,7 @@ const path = require('path');
 
 module.exports = {
   stories: ['../src/**/*.story.mdx', '../src/**/*.story.@(js|jsx|ts|tsx)'],
-  staticDirs: ["../public"],
+  staticDirs: ['../public'],
   addons: [
     '@storybook/addon-links',
     '@storybook/addon-essentials',
@@ -33,6 +33,10 @@ module.exports = {
       ],
       include: path.resolve(__dirname, '../'),
     });
+    config.resolve.alias = {
+      ...config.resolve.alias,
+      '@': path.resolve(__dirname, '../src'),
+    };
     return config;
   },
   framework: '@storybook/react',

